squad-server/plugins/auto-tk-warn.js

```diff
@@ -34,8 +44,11 @@ export default class AutoTKWarn extends BasePlugin {
   }
 
   async onTeamkill(info) {
-    if (!info.attacker) return;
-
-    await this.server.rcon.warn(info.attacker.steamID, this.options.message);
+    if (info.attacker) {
+      this.server.rcon.warn(info.attacker.steamID, this.options.attacker_message);
+    }
+    if (this.options.warn_victim && info.victim) {
```

Comment:
We can probably do away with the `warn_victim` and just do it based on whether the message is defined or not - which means they do not need to be required.

```js
if (info.attacker && this.options.attackerMessage) {
  await this.server.rcon.warn(info.attackers.steamId, this.options.attackerMessage);
}

if (info.victim && this.options.victimMessage) {
  await this.server.rcon.warn(info.attackers.steamId, this.options.victimMessage);
}
```

Reply:
Yeah, that's true, we can probably do without them.